util/tasks.go

```diff
@@ -96,6 +98,35 @@ func Reset() {
 	}
 }
 
+func Web(url string){
+	cmd := exec.Command("docker-compose", "up -d --build")
+	cmd.Dir = "web"
```

Comment:
Can we update this workflow to git clone or install "oauth2l-web" locally if it doesn't exist? OAuth2l will be released as a binary, and all dependencies must be installed separately as a pre-requisite. Installation path of oauth2l-web could be a configurable option for Web.